go/vt/automation/testing_emit_echo_task.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, Google Inc. All rights reserved.
```

Comment:
I don't see the value in this being its own file - combine all testing tasks into one file?